src/vm/loaderallocator.hpp

```diff
@@ -485,7 +485,6 @@ typedef VPTR(LoaderAllocator) PTR_LoaderAllocator;
 class GlobalLoaderAllocator : public LoaderAllocator
 {
     VPTR_VTABLE_CLASS(GlobalLoaderAllocator, LoaderAllocator)
-    VPTR_UNIQUE_OVERRIDE(VPTRU_LoaderAllocator+1)
```

Comment:
I'm gonna try maybe to add a `no-inconsistent-missing-override` flag specifically for OSX and reintroduce overrides there just to check.